### DISCUSSION TITLE
Can I separate the path of ssd and hdd in filer

### DISCUSSION THREAD

**Discussion by BakerBunker** [seaweedfs/seaweedfs#4348] [Q&A] [answered]
Now I want to deploy a seaweed service on a cluster and currently need to differentiate the path of ssds and hdds. Do I need to deploy two sets of master-volume-filer or can I achieve it with just one set?

**Comment by aoberest** [comment, marked as answer]
Hi. May be you need this https://github.com/seaweedfs/seaweedfs/wiki/Tiered-Storage#ssd--hdd-volume-disk-types 
you can to add tags for volumes
